RileyLink/DeviceDataManager.swift

```diff
@@ -242,46 +370,45 @@ class DeviceDataManager {
 
     init() {
```

Comment:
Sorry to burden you with the haphazard expansion of Loop's DDM. I promise this will be refactored as things settle.